README.md

```diff
@@ -1 +1,360 @@
+
+```
+ “qrCode”: “cp_f551e0d9-5d50-47b6-bb77-faba277a6042", 
+ “requestCode”: “9VHg2V1QR7a7d/q6J3pgQg==“, 
+ “status”: “new”, 
+ “createdAt”: “2020-02-26T00:13:56.071Z”, 
+ “updatedAt”: “2020-02-26T00:13:56.071Z” 
+}
+```
+
+<br>
+
+### [Getting the information about a payment request](https://service.centrapay.com/payments/api/documentation#/Requests/getRequestsinfo)
+
+A payment request can be created by sending a **GET** request to the following endpoint:<br>
+https://service.centrapay.com/payments/api/requests.info
+
+The following parameters **must** be provided with the request query:
+
+| Parameter | Description                                                               |
+|-----------|---------------------------------------------------------------------------|
+| requestId | The payment requestId that is generated when `/requests.create` is called |
+
+>Entering the requestId in `/requests.info` will return information about the request and its status.
+
+<br>
+
+### [Cancelling a payment request](https://service.centrapay.com/payments/api/documentation#/Requests/postRequestscancel)
+
+A payment request can be created by sending a **POST** request to the following endpoint:<br>
+https://service.centrapay.com/payments/api/requests.cancel 
+
+The following parameters **must** be provided with the request query:
+
+| Parameter | Description                                                               |
+|-----------|---------------------------------------------------------------------------|
+| requestId | The payment requestId that is generated when `/requests.create` is called |
+>Entering the requestId in `/requests.cancel` will allow you to cancel a request.
+You are unable to cancel a request where its status has been set to paid.
+
+<br>
+
+### [Voiding a payment request](https://service.centrapay.com/payments/api/documentation#/Requests/postRequestsvoid)
+
+A payment request can be created by sending a **POST** request to the following endpoint:<br>
+https://service.centrapay.com/payments/api/requests.void
+
+The following parameters **must** be provided with the request query:
+
+| Parameter | Description                                                                                                                                                                                                                                                             |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| requestId | The payment requestId that is generated when `/requests.create` is called. Entering the requestId in `/requests.info` will allow you to determine what state the request is in and cancel the request if it hasn’t been paid or refund the request if it has been paid. |
```

Comment:
Entering the requestId in `/requests.info` will allow you to determine what state the request is in and cancel the request if it hasn’t been paid or refund the request if it has been paid. <--- this part isn't needed as we handle that on the backend for them